public/locales/pt/components.json

```diff
@@ -0,0 +1,179 @@
+{
+  "additembutton": {
```

Comment:
The environment is not working as @fonzy911 mentioned cause the keys are not resolving. All of them were "lowercase-ed". For example this one should be `addItemButton`, not `additembutton` 👍